db/migrate/20200409033412_create_bookmarks_from_post_action_bookmarks.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+class CreateBookmarksFromPostActionBookmarks < ActiveRecord::Migration[6.0]
+  def up
+    SiteSetting.enable_bookmarks_with_reminders = true
+
+    post_action_bookmarks = PostAction
```

Comment:
Can you use manual SQL here? Using AR in migrations leads to many problems down the road.

Reply:
@eviltrout you are right, I wrote this as a rake task originally and moved it into a migration but didn't consider the AR issue. I will change this.